docs/exchange-operators/rosetta/run-with-docker.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Running with Docker
+description: Run Mina’s implementation of Rosetta API with Docker.
+keywords:
+  - historical data
+  - blockchain integration
+  - explorer
+  - wallet
+  - archive node
+  - graphQL
+  - rosetta
+  - docker
+---
+
+
+# Running with Docker
+
+:::info
+
+There is a known issue that you can't use the docker image on the Apple silicon chip.
+To run Mina Rosetta on Apple silicon, you can use the steps in [Building and running Rosetta from source code](build-from-sources).
```

Comment:
This is being dealt with by @joaosreis as we speak. Please consult him for an update on this issue.